Source/JavaScriptCore/wasm/WasmTypeDefinition.h

```diff
@@ -904,7 +918,7 @@ class TypeInformation {
     RefPtr<TypeDefinition> m_I32_I32;
     RefPtr<TypeDefinition> m_I32_RefI32I32;
     RefPtr<TypeDefinition> m_Ref_RefI32I32;
-    RefPtr<TypeDefinition> m_Ref_I32I32I32I32;
+    RefPtr<TypeDefinition> m_Arrayref_I32I32I32I32;
```

Comment:
Note: the style checker complains on this but the name is consistent with existing code.